src/scripts/main.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// write your code here
+const collection = document.querySelectorAll('[class="population"]');
```

Comment:
Instead of querying for all elements with class 'population' .population'

Reply:
ok, thanks